Formula/pnpm.rb

```diff
@@ -23,12 +23,39 @@ class Pnpm < Formula
   depends_on "node"
 
   def install
+    (prefix/"etc").mkpath
+    (prefix/"etc/npmrc").atomic_write "global-bin-dir = ${HOME}/Library/pnpm\n" if OS.mac?
+    (prefix/"etc/npmrc").atomic_write "global-bin-dir = ${HOME}/.local/pnpm\n" if OS.linux?
```

Comment:
```suggestion
    npmrc = if OS.mac? ? "global-bin-dir = ${HOME}/Library/pnpm\n" : "global-bin-dir = ${HOME}/.local/pnpm\n"
    (prefix/"etc/npmrc").write npmrc
```
Writing an `npmrc` here will cause formula linking issues for anyone with a `$HOMEBREW_PREFIX/etc/npmrc` (which on macOS is `/usr/local/etc/npmrc`, so I don't imagine this will be too uncommon).

Are there any other solutions we can use?

Reply:
Once I proposed `/usr/local/etc/pnpmrc` to avoid confliction but @zkochan liked `/usr/local/etc/npmrc` as it is the standard of npm.
@zkochan What do you think about this?

Reply:
Currently we use npm's configuration system, so it should be where the npm's config file would be. And it should have the npm configuration name.

Reply:
@zkochan What about a built-in config file dedicated for pnpm?

https://docs.npmjs.com/cli/v7/configuring-npm/npmrc#built-in-config-file

Reply:
> Currently we use npm's configuration system, so it should be where the npm's config file would be. And it should have the npm configuration name.

This makes it difficult to provide custom config files for packages without inadvertently stomping over a user's config.

Reply:
I think what we need is a way to provide a Homebrew's custom configuration file only for pnpm not for npm properly.